rosdep/python.yaml

```diff
@@ -9209,6 +9209,56 @@ python3-sparkfun-ublox-gps-pip:
   ubuntu:
     pip:
       packages: [sparkfun-ublox-gps]
+python3-spconv-pip:
+  debian:
+    pip:
+      packages: [spconv]
+  fedora:
+    pip:
+      packages: [spconv]
+  ubuntu:
+    pip:
+      packages: [spconv]
+python3-spconv-cuda-114-pip:
```

Comment:
could you use the pip name?
i.e. `python3-spconv-cu114-pip`

Reply:
Sorry for the delay. Applied!